plugins/catalog/src/components/CatalogPage/DefaultCatalogPage.tsx

```diff
@@ -70,9 +76,13 @@ export function DefaultCatalogPage(props: DefaultCatalogPageProps) {
   const createComponentLink = useRouteRef(createComponentRouteRef);
 
   const { createButtonTitle } = useCatalogPluginOptions();
+  const { t } = usePluginTranslation(catalogTranslationRef);
 
   return (
-    <PageWithHeader title={`${orgName} Catalog`} themeId="home">
+    <PageWithHeader
+      title={`${orgName} ${t('catalog', 'Catalog')}`}
```

Comment:
i18next support some form of templating right? so that this could be expressed something like this instead

```ts
t('catalog', {
  defaultMessage: '{orgName} Catalog',
  values: { orgName },
})
```